bentoml/_internal/log.py

```diff
@@ -9,25 +11,32 @@
 from .trace import ServiceContext
 from .configuration import get_debug_mode
 
+TRACED_LOG_FORMAT = "[%(component)s] %(message)s (trace=%(trace_id)s,span=%(span_id)s,sampled=%(sampled)s)"
+SERVICE_LOG_FORMAT = "[%(component)s] %(message)s"
+DATE_FORMAT = "%x %X"
+
 
 class TraceFilter(Filter):
     """
     Logging filter implementation that injects tracing related fields to the log record.
     """
 
-    def filter(self, record):
-        record.sampled = ServiceContext.sampled
-        record.trace_id = ServiceContext.trace_id
-        record.span_id = ServiceContext.span_id
-        record.request_id = ServiceContext.request_id
-        record.component = ServiceContext.component_name
-
-        return Filter.filter(self, record)
+    def filter(self, record: logging.LogRecord) -> bool:
+        if "circus" in record.name:
+            component_name = "circus"
+        elif "asyncio" in record.name:
+            component_name = "asyncio"
+        else:
+            component_name = ServiceContext.component_name
```

Comment:
Not a blocker but why do we want `circus` or `asyncio` as the component name, as opposed to using the actual component as the name?

Reply:
I was thinking to separate these logs out, better separation and helps us know where the source of the logs come from.